.github/workflows/e2e-testbench.yaml

```diff
@@ -35,15 +73,16 @@ jobs:
         \"channel\": \"Internal Dev\",
         \"branch\": \"${{ inputs.branch || 'main' }}\",
         \"build\":  \"${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}\",
-        \"clusterPlan\":\"Production - M\",
+        \"clusterPlan\":\"${{ inputs.clusterPlan }}\",
```

Comment:
:question: I guess this may have been answered in the previous PR, but why do we have to escape all the quotes here? AFAIK with multi-line YAML the quotes would be preserved in the output, no? Then when using the input, you could `toJSON(inputs.variables)` to properly quote/escape it? :thinking: Anyway, out of scope, just curious

Reply:
I don't know if it works. I went the known way because it is so time consuming to test it :D I could give your suggestion a try. This escaping was giving me headache anyway.

Reply:
I gave it a try https://github.com/camunda/zeebe/actions/runs/4073651446/jobs/7017830750 It worked partially, but there are some issues. I do not want to spend more time on it. So I will leave it as it is.